src/app/components/PlayerPanel.ts

```diff
@@ -109,37 +109,85 @@ export class PlayerPanel {
           'player-panel__image',
         ),
         `x${count}`,
-      ),
+      ),*/
+  // );
+  // }
+  private createResultTitle(): HTMLElement {
+    return Render.elementFactory(
+      'p',
+      { className: 'player-panel__title' },
+      'Results',
     );
   }
-
   private createResultWindow(): HTMLElement {
-    return Render.elementFactory(
-      'div',
-      {
-        className: 'player-panel__result',
-      },
-      'Results:',
-    );
+    return Render.elementFactory('div', {
+      className: 'player-panel__result',
+    });
   }
 
-  private createButtonPanel(): HTMLElement {
-    return Render.elementFactory(
-      'div',
-      { className: 'player-panel__buttons' },
-      this.createDiceButton(),
-      this.createExchangeButton(),
+  // private createButtonPanel(player: Player): HTMLElement {
+  // return Render.elementFactory(
+  // 'div',
+  // { className: 'player-panel__buttons' },
+  // this.createExchangeButton(player),
+  // this.createDiceButton(player),
+  // );
+  // }
+  private createExchangeButton(player: Player): HTMLElement {
+    const tradeBtn = Render.elementFactory(
+      'button',
+      {
+        id: 'exchange',
+        className: 'btn button',
+        style: `border:1px solid ${player.theColor}`,
+      },
+      Render.elementFactory('img', {
+        className: 'btn__img',
+        src: 'static/images/ui/vector.png',
+      }),
+      Render.elementFactory(
+        'div',
+        { className: 'btn__text' },
+        Render.elementFactory(
+          'p',
+          { className: 'btn__text--big' },
+          'Exchange',
+        ),
+        Render.elementFactory(
+          'p',
+          { className: 'btn__text--small' },
+          'animals',
+        ),
+      ),
     );
+    tradeBtn.addEventListener('click', () => this.view.handleTrade());
+    return tradeBtn;
   }
-
-  private createDiceButton(): HTMLElement {
+  private createDiceButton(player: Player): HTMLElement {
     const rollBtn = Render.elementFactory(
       'button',
       {
         id: 'roll-dice',
         className: 'btn button',
+        style: `border:1px solid ${player.theColor}`,
```

Comment:
Like exchange button, move border-width and border-style to button__panel class (or panel__button?)